io-engine/src/bdev/null_ng.rs

```diff
@@ -16,11 +16,15 @@ use spdk_rs::{
 const NULL_MODULE_NAME: &str = "NullNg";
 
 /// Poller data for Null Bdev.
+#[derive(Default)]
+#[allow(clippy::non_send_fields_in_send_ty)]
 struct NullIoPollerData<'a> {
     iovs: RefCell<Vec<BdevIo<NullIoDevice<'a>>>>,
     _my_num: f64,
 }
 
+unsafe impl<'a> Send for NullIoPollerData<'a> {}
```

Comment:
should add a comment static why this is ok?

Reply:
That struct contains RefCell, which is not Send. It is okay to "force" it to be Send, because nothing "unsendable" is done with this struct.
It's an example code, not actually used one.
But in real code, RefCell + Send would be okay for many cases when one needs a mutable poller data: a poller is always polled from its own thread, be it the master or a separate thread. (Unless the data is accessed elsewhere besides the poller).

Reply:
I will fix comment.

Reply:
I'm wrong, NullIoPollerData.iovs is accessed elsewhere. It must be a mutex here, not RefCell. I will fix it.